### DISCUSSION TITLE
Developer documentation?

### DISCUSSION THREAD

**Discussion by andrewpros** [openclassify/openclassify#1338] [General]
Hi.

Looks like a great project, but i cannot find any developer documentation, is there any, i like the "modular and most advanced open source classified platform" and all, but without docs, that does not mean much.

**Comment by andrewpros** [comment]
This is still a concern, but seems there is zero dev docs.

**Reply by erhanask** [reply]
Hey!

We are working on a detailed documentation and also we are sorry about delay for answer.

You can check this link 
https://visiosoft.gitbook.io/v2/

**Reply by andrewpros** [reply]
Wow, didn't expect that after so long since my first ask, im positively surprised, there is a light in the tunnel, thank you very much.

**Comment by agenthinson99** [comment]
![aa](https://github.com/user-attachments/assets/eaa9060c-edd2-4c37-998a-498560904bd7)
We offer all types of financial loans with an interest rate of 2%. Our offer ranges from a minimum amount of $10,000.00 USD up to a maximum amount of $10,000,000.00 USD for individuals & business purposes. We offer a wide range of financial services which includes: Business planning, commercial and development finance, properties and mortgages, debt consolidation loans, business loans, private loans, home refinancing, hotel loans, student loans etc. Kindly respond immediately via email below for more information.

Lender's Email: agenthinson@gmail.com

Interested applicants should apply with the following information below:-

Name:
Country:
Loan Amount: 
Duration:
WhatsApp Phone#:
Gender:

**Reply by agenthinson99** [reply]
We offer all types of financial loans with an interest rate of 2%. Our offer ranges from a minimum amount of $10,000.00 USD up to a maximum amount of $10,000,000.00 USD for individuals & business purposes. We offer a wide range of financial services which includes: Business planning, commercial and development finance, properties and mortgages, debt consolidation loans, business loans, private loans, home refinancing, hotel loans, student loans etc. Kindly respond immediately via email below for more information.

Lender's Email: agenthinson@gmail.com

Interested applicants should apply with the following information below:-

Name:
Country:
Loan Amount: 
Duration:
WhatsApp Phone#:
Gender:
![aa](https://github.com/user-attachments/assets/e1ac3330-7b60-4391-a71b-aeeea69a26ce)
